reference/controller.md

```diff
@@ -7,29 +7,27 @@
 
 (controller-tag)=
 ## Controller tag
-> See first: {ref}`tag`
 
 A controller tag has the following format:
 
 ```text
 controller-<controller name>
 ```
 
-(controller-relation)=
-## Controller relation
+(controller-permission)=
+## Controller permission
```

Comment:
One to chew on. Does this read better as "Controller permission" or "Controller access". I think they could be used interchangeably depending on which is preferred.

Reply:
I generally find "access" more intuitive. Do you want me to look into switching to "access" everywhere?